modules/virt-removing-interface-from-nodes.adoc

```diff
@@ -5,8 +5,10 @@
 [id="virt-removing-interface-from-nodes_{context}"]
 = Removing an interface from nodes
 
-Remove an interface from nodes by editing the `NodeNetworkConfigurationPolicy` object and set
-the `state` of the interface to `absent`.
+Remove an interface from nodes by editing the `NodeNetworkConfigurationPolicy` object and set the `state` of the interface to `absent`.
+
+Removing an interface from a node does not automatically restore the node network configuration to a previous state. After you remove an interface, any of the node NICs throughout the cluster that were previously attached or subordinate to the interface are placed in a `down` state and become unreachable. +
```

Comment:
The need to bring eth0 up again applies only when it was attached to a bridge or a bonding. Maybe we could change:

"subordinate to the interface" -> "subordinate to a bridge or a bonding"

Reply:
Updated as suggested. Do you think we need to be more explicit about the difference between bridge & bond vs vlan interface or do you think this is sufficient